src/meta/meta_data.cpp

```diff
@@ -479,6 +479,37 @@ void app_state_helper::on_init_partitions()
     restore_states.resize(owner->partition_count);
 }
 
+void app_state_helper::reset_manual_compact_status()
+{
+    for (auto &cc : contexts) {
```

Comment:
for (const auto &cc : contexts) {

Reply:
This for sentence will update value of `contexts`, it is not a read-only iterator, so it can't be const.